src/identity.ts

```diff
@@ -148,6 +148,64 @@ export const removeTrustedAuthority = async (authorityId: string, options: { ide
     }
 };
 
+export const parseInput = (fileName?: string) => {
+    let data = undefined;
+    if (!fileName) {
+        data = JSON.parse(fs.readFileSync(0, 'utf-8'));
+    }
+    else {
+        if (!fs.existsSync(fileName)) {
+            console.log(chalk.bold.red("The credential file does not exist."));
+            return;
+        }
+        const file = fs.readFileSync(fileName, 'utf8');
+        data = JSON.parse(file);
+    }
+    return data;
+}
+
+export const createCredential = async (options: { identityFile: string; did: string, credential?: string }) => {
+    try {
+        const { identityFile, did, credential } = options;
+        let credentialData = parseInput(credential);
+        const { credentialType, claimType, claim } = credentialData;
+        let error = false;
+        if (!credentialType) {
+            console.log(chalk.bold.red("credentialType not present in credential file"));
+        }
+        if (!claimType) {
+            console.log(chalk.bold.red("claimType not present in credential file"));
+        }
+        if (!claim) {
+            console.log(chalk.bold.red("claim not present in credential file"));
+        }
+        if (error) {
```

Comment:
this is always false so it can be removed

Reply:
missing "error = true"